xgress/link_send_buffer.go

```diff
@@ -103,7 +103,7 @@ func NewLinkSendBuffer(x *Xgress) *LinkSendBuffer {
 		x:                 x,
 		buffer:            make(map[int32]*txPayload),
 		newlyBuffered:     make(chan *txPayload),
-		newlyReceivedAcks: make(chan *Acknowledgement, 2),
+		newlyReceivedAcks: make(chan *Acknowledgement, 4),
```

Comment:
Why 2 and then why 4?

Reply:
Testing showed lots of drops of acks, so I bumped this up to give it a bit more space. Testing didn't show much difference.